### DISCUSSION TITLE
Welcome to Hakchi2-CE Discussions!

### DISCUSSION THREAD

**Discussion by DanTheMan827** [TeamShinkansen/Hakchi2-CE#422] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by taker24** [comment]
Hi,
I've installed hakchi2 on my snes mini with retroarch 1.8.4. I try to launch DBZ hyperdimention, DBZ Sayen legend, BS Zelda no denetsu (with argument --retroarch), and mighty morfin power ranger the movie (megaadrive verion) and each time I try to lauch game with retroarch I have a black screen during 4-5 sec a It return to main menu. Can you help me please?

**Comment by KMFDManic** [comment]
The --retroarch doesnt work.  In newer hakchi, youd need to have an SNES Core installed along with retroarch.  Then, right click, select emulation core, resynch, and u should be fine.

**Comment by ghbbeep** [comment]
Where can I put my SatellaWave files?

**Comment by GFBmsoft** [comment]
Hi, support for Super mini sfc ?
